packages/genesys-spark-tokens/data/semantic/theme/flare.json

```diff
@@ -59,15 +59,15 @@
             "type": "color"
           },
           "40d": {
-            "value": "{color.azure_blue.1000}",
+            "value": "{color.haze.850}",
```

Comment:
was this color meant to change ?

Reply:
Yes, we change the color of the secondary button from blue to gray-ish in dark mode